src/NBomber/Infra/Reporting.fs

```diff
@@ -11,20 +11,29 @@ open NBomber.Contracts
 open NBomber.Domain
 open NBomber.Statistics
 
-let buildReport (scenario: Scenario, flowInfo: FlowInfo[]) =        
+let buildReport (scenario: Scenario, flowInfo: FlowInfo[]) = 
+    let getPausedTime step =
+        match step with Pause time -> time.Ticks | _ -> int64 0
```

Comment:
try to not use match expression in one line, it probably does make sense only in case with |> Arrray.map